test-requirements.txt

```diff
@@ -1,33 +1,34 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.8
+# by the following command:
```

Comment:
What version of pip-tools did you use to generate this file?

This requirements file installs `pip-tools==6.2.0`.  If you used a different version, it might cause formatting changes like this.

Reply:
I was using `pip-tools==7.4.1`. I've checked the related packages and updated the test-requirements.txt. [62dca6b](https://github.com/groove-x/trio-util/pull/26/commits/62dca6bb265f3c21b9296404d0d19e5bfe587b00)